dist/ba_root/mods/games/VolleyBall.py

```diff
@@ -535,3 +526,256 @@ def _spawn_puck(self) -> None:
+                    'color':(1,0,0),'opacity':1,'draw_beauty':True,'additive':False,'size':[0.40]})
+            self.zone = ba.newnode('locator',attrs={'shape':'circle','position':(11,0.01,-z),
+                    'color':(1,0,0),'opacity':1,'draw_beauty':True,'additive':False,'size':[0.40]})
+            self.zone = ba.newnode('locator',attrs={'shape':'circle','position':(-11,0.01,z),
+                    'color':(0,0,1),'opacity':1,'draw_beauty':True,'additive':False,'size':[0.40]})
+            self.zone = ba.newnode('locator',attrs={'shape':'circle','position':(-11,0.01,-z),
+                    'color':(0,0,1),'opacity':1,'draw_beauty':True,'additive':False,'size':[0.40]})
+            z+=1
+
+        self.node = ba.newnode('terrain',
+                               delegate=self,
+                               attrs={
+                                   'model':
+                                       None,
+                                   'collide_model':
+                                       ba.getcollidemodel('footballStadiumCollide'), # we dont want Goalposts...
+                                   'color_texture':
+                                       self.preloaddata['tex'],
+                                   'materials': [
+                                       shared.footing_material]
+                               })
+        ba.newnode('terrain',
+                   attrs={
+                       'model': self.preloaddata['vr_fill_model'],
+                       'vr_only': True,
+                       'lighting': False,
+                       'background': True,
+                   })
+        mats = [shared.footing_material]
+        self.floor = ba.newnode('terrain',
+                                attrs={
+                                    'model': self.preloaddata['models'][1],
+                                    'color_texture': self.preloaddata['tex'],
+                                    'opacity': 0.92,
+                                    'opacity_in_low_or_medium_quality': 1.0,
+                                    'materials': mats,
+                                    'color': (0.4,0.9,0)
+                                })
+
+        self.background = ba.newnode(
+            'terrain',
+            attrs={
+                'model': ba.getmodel('natureBackground'),
+                'lighting': False,
+                'background': True,
+                'color': (0.5,0.30,0.4)
+            })
+
+        gnode = ba.getactivity().globalsnode
+        gnode.floor_reflection = True
+        gnode.debris_friction = 0.3
+        gnode.debris_kill_height = -0.3
+        gnode.tint = (1.2, 1.3, 1.33)
+        gnode.ambient_color = (1.15, 1.25, 1.6)
+        gnode.vignette_outer = (0.66, 0.67, 0.73)
+        gnode.vignette_inner = (0.93, 0.93, 0.95)
+        gnode.vr_camera_offset = (0, -0.8, -1.1)
+        gnode.vr_near_clip = 0.5
+        #self.is_hockey = True
+
+
+
+
+ba._map.register_map(VolleyBallMap)
```

Comment:
same here , need to remove map

Reply:
Does it really matter? I've removed plugin to register maps so it works just fine even if placed in games folder??